### PR TITLE
add hta page

### PR DESCRIPTION
**Pull request by hoppanda**

### CODE REVIEW

**Review by danielinteractive — COMMENTED** [review]
Great, thanks Greg! Can you please also add a News entry for this ?

File: data/members.csv [hunk 1]
```diff
@@ -1,37 +1,42 @@
...
+Laura,Harris,Denali,1,0,0
+Colombe,Chappey,Pfizer,1,0,0
+Martin,Kappler,Cytel,1,0,0
+Pravin,Madhavan,Berry Consultants,1,0,0
+Nick,Berry,Berry Consultants,1,0,0
+Issac,Gravestock,0,0,1
+Christos,Kokaliaris,0,0,1
+Michael,Seo,0,0,1
+Miranta,Antoniou,0,0,1
+Rossella,Belleli,0,0,1
```
Comment (danielinteractive) [inline data/members.csv:42]:
```suggestion
Rossella,Belleli,0,0,1
Quang,Nguyen,1,0,0
```
just adding a new member here

File: hta_page.qmd [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: "HTA-R workstream"
+---
+
+**Lead**: Gregory Chen (MSD)
+
+**Objective**: 
+This workstream in SWE WG unites voluntary statistical software engineers that are experienced in HTA (Health Technology Assessment) or highly-interested in this area. 
+The team aims to develop open-source R tools of good quality in the right formats (R pkg, apps, user guides) that are useful to both manufacturers and payers, to support crucial analytic topics in HTA dossier submission across various countries, particularly the topics with unmet needs in R implementation and/or related to [EUnetHTA](https://www.eunethta.eu/).
```
Comment (ellessenne) [inline hta_page.qmd:9]: I would suggest not abbreviating "packages" with "pkg" here.

File: hta_page.qmd [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+The workstream also works within a broader group (link tbd) of business and methodological SMEs in HTA and Market Access space of our pharma industry, to ensure this open-source HTA pipeline of R tools is relevant over time and the tools can be adopted easily by business and payers.
```
Comment (ellessenne) [inline hta_page.qmd:11]: I think there is no need to refer to a "link tbd" if that doesn't exist – can always be added later?

File: hta_page.qmd [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+**Lead**: Gregory Chen (MSD)
+
+**Objective**: 
+This workstream in SWE WG unites voluntary statistical software engineers that are experienced in HTA (Health Technology Assessment) or highly-interested in this area. 
+The team aims to develop open-source R tools of good quality in the right formats (R pkg, apps, user guides) that are useful to both manufacturers and payers, to support crucial analytic topics in HTA dossier submission across various countries, particularly the topics with unmet needs in R implementation and/or related to [EUnetHTA](https://www.eunethta.eu/).
+
+The workstream also works within a broader group (link tbd) of business and methodological SMEs in HTA and Market Access space of our pharma industry, to ensure this open-source HTA pipeline of R tools is relevant over time and the tools can be adopted easily by business and payers.   
+
+Current Work Plan: link tbd
```
Comment (ellessenne) [inline hta_page.qmd:13]: Same here with the "tbd" link.

**Review by ellessenne — CHANGES_REQUESTED** [review]
Hi,
Looks good overall! Just a couple of minor points in the page for the working group.
Thanks,

Alessandro

File: hta_page.qmd (hunk 3 above)
Comment (danielinteractive) [reply hta_page.qmd:11]: Yeah I would agree. Also from strategy perspective, it might be easier to think about collaboration SWE WG - HTA WGs rather than this workstream being wrapped into yet another group?

**Comment by danielinteractive** [comment]
Thanks @hoppanda for the PR, could you maybe address the comments this week so that we can merge it soon? thank you!

File: data/members.csv [hunk 5]
```diff
@@ -1,37 +1,43 @@
...
+Quang,Nguyen,1,0,0
```
Comment (danielinteractive) [inline data/members.csv:43]:
```suggestion
```
decided to do this in separate PR

File: data/members.csv (hunk 1 above)
Comment (danielinteractive) [inline data/members.csv:42]:
```suggestion
Issac,Gravestock,Roche,0,0,1
Christos,Kokaliaris,Roche,0,0,1
Michael,Seo,Roche,0,0,1
Miranta,Antoniou,Roche,0,0,1
Rossella,Belleli,Roche,0,0,1
```

Comment (hoppanda) [reply data/members.csv:42]: adopted

File: hta_page.qmd (hunk 2 above)
Comment (hoppanda) [reply hta_page.qmd:9]: adopted

File: hta_page.qmd (hunk 3 above)
Comment (hoppanda) [reply hta_page.qmd:11]: link tbd is deleted. We can discuss more about the phrasing about the connection between this wg with the broader group. The broader group has a meeting at March 8.

File: hta_page.qmd (hunk 4 above)
Comment (hoppanda) [reply hta_page.qmd:13]: adopted

**Comment by hoppanda** [comment]
@danielinteractive @ellessenne  please review. Have merged the update from Master branch, if okay, pls merge.

File: hta_page.qmd [hunk 6]
```diff
@@ -0,0 +1,21 @@
+---
+title: "HTA-R workstream"
+---
+
+**Lead**: Gregory Chen (MSD)
+
+**Objective**: 
+This workstream in SWE WG unites voluntary statistical software engineers that are experienced in HTA (Health Technology Assessment) or highly-interested in this area.
```
Comment (danielinteractive) [inline hta_page.qmd:8]:
```suggestion
This workstream of the SWE WG unites voluntary statistical software engineers that are experienced in HTA (Health Technology Assessment) or highly-interested in this area. 
```

File: hta_page.qmd [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+**Lead**: Gregory Chen (MSD)
+
+**Objective**: 
+This workstream in SWE WG unites voluntary statistical software engineers that are experienced in HTA (Health Technology Assessment) or highly-interested in this area. 
+The team aims to develop open-source R tools of good quality in the right formats (R packages, apps, user guides) that are useful to both manufacturers and payers, to support crucial analytic topics in HTA dossier submission across various countries, particularly the topics with unmet needs in R implementation and/or related to [EUnetHTA](https://www.eunethta.eu/).
+
+The workstream also works within a broader group of business and methodological SMEs in HTA and Market Access space of our pharma industry, to ensure this open-source HTA pipeline of R tools is relevant over time and the tools can be adopted easily by business and payers.   
+
+Github repo: [link](https://www.github.io/hta-pharma/).
```
Comment (danielinteractive) [inline hta_page.qmd:13]:
```suggestion
Find more details in our GitHub organization: [link](https://www.github.io/hta-pharma/).
```

File: hta_page.qmd [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+
+```{r, label = "active", echo = FALSE, message = FALSE, warning = FALSE, results = "asis"}
+path <- getwd()
+source(file = paste(path, "/utils/read_members.R", sep = ""))
+read.members("HTA")
+
```
Comment (danielinteractive) [inline hta_page.qmd:20]:
```suggestion
```

File: data/news.csv [hunk 9]
```diff
@@ -1,4 +1,5 @@
 Date,Text
+02/25/2023,<a href=‘./hta_page.html’ title=‘technology icons’>HTA Workstream is here now</a>
```
Comment (danielinteractive) [inline data/news.csv:2]:
```suggestion
03/05/2023,<a href=‘./hta_page.html’ title=‘technology icons’>HTA Workstream is here now</a>
```

**Review by danielinteractive — APPROVED** [review]
thanks @hoppanda , good to merge after including few polishing comments inside

File: hta_page.qmd (hunk 8 above)
Comment (hoppanda) [reply hta_page.qmd:20]: @danielinteractive  without this part, will there still be a list of HTA work stream members displayed?

Comment (danielinteractive) [reply hta_page.qmd:20]: I would just delete the blank line of code, not any actual code here

Comment (hoppanda) [reply hta_page.qmd:20]: got it! thx

**Comment by hoppanda** [comment]
@danielinteractive @ellessenne  all comments are adopted, pls help to merge

**Review by danielinteractive — APPROVED** [review]
